File: Java/FiboEuler2.java
public class FiboEuler2
{
  public static void main(String[] args)
  {
    int value1 = 1;
    int value2 = 1;
    final int LASTVALUE = 4000000;
    int total = 0;

    while(value2 < LASTVALUE)
    {
      int nextValue = value1 + value2;
      if(nextValue % 2 ==0)
      {
        total = total + nextValue;
      }
      value1= value2;
      value2 = nextValue;
    }
    System.out.println("The sum of Fibonacci numbers that are less than 4,000,000 is: " + total);
  }
}

File: Python/FiboEuler2.py
a = 1
b = 1
c = 0
finalNum = 4000000
total = 0
while c < finalNum:
    c = a + b
    if c % 2 == 0:
        total = total + c
    a = b
    b = c
print "Total of even Fibonacci numbers less than 4,000,000: %d" % total
